{"name": "Негативный тест авторизации существующего пользователя", "status": "passed", "description": "Негативный тест. Получение  в тело ответа. code == 403 и наличия в ответе кода \"message\": \"User already exists\"", "start": 1728531663499, "stop": 1728531664547, "uuid": "ab31e931-520a-4a69-a961-69833830fbcd", "historyId": "10cfcc20049614abd6aceb6fc39f273e", "testCaseId": "10cfcc20049614abd6aceb6fc39f273e", "fullName": "test_registration.TestRegistration#test_repeat_registration_of_the_existing_user", "labels": [{"name": "suite", "value": "test_registration"}, {"name": "subSuite", "value": "TestRegistration"}, {"name": "host", "value": "LAPTOP-KL5MP4V8"}, {"name": "thread", "value": "14656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_registration"}]}